docs/Running-Tachyon-with-RDMA.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: global
+title: Running Tachyon with RDMA over JXIO
+---
+
+## Prerequisite
+
+* **Compile JXIO**. Download and compile [JXIO](https://github.com/accelio/JXIO/).
```

Comment:
currently no.
jxio is using native components thus needs to be compiled for specific targets